src/AddAssessment.js

```diff
@@ -22,9 +22,27 @@ class AddAssessment extends Component {
     this.setState(newState);
   }
 
+  addAssessment () {
+    let init = {
+      method: 'POST',
+      body: JSON.stringify(Object.assign({}, this.state, {user_id: "TEST"})) // TODO: replace with real user id
+    };
+
+    fetch(`${process.env.REACT_APP_API_URL}/assessment`, init)
```

Comment:
Just a comment around extracting this logic into it's own file so that we can keep this file size down :)

Reply:
:+1: Sounds good. Do you mean abstracting the fetch logic in general, or just extracting this specific addAssessment function?

Reply:
the fetch logic. :)